docs/roman/pipeline_run.rst

```diff
@@ -95,13 +94,11 @@ For the mosaic level pipeline and steps,
 ::
 
  from romancal.pipeline import MosaicPipeline
- result = ExposurePipeline.call('r0000101001001001001_asn.json')
+ mosp = MosaicPipeline('r0000101001001001001_asn.json')
 
  from romancal.skymatch import SkyMatchStep
- result = SkyMatchStep.call('r0000101001001001001_asn.json')
-
+ skymatch = SkyMatchStep.call('r0000101001001001001_asn.json')
```

Comment:
```suggestion
result = SkyMatchStep.call('r0000101001001001001_asn.json')
```
Assigning to `skymatch` is misleading as `call` is not returning a `SkyMatchStep` instance.